src/expr/Cargo.toml

```diff
@@ -11,56 +11,56 @@ name = "like_pattern"
 harness = false
 
 [dependencies]
-aho-corasick = "0.7.20"
-anyhow = "1.0.66"
-bytes = "1.3.0"
-bytesize = "1.1.0"
-chrono = { version = "0.4.23", default-features = false, features = ["std"] }
-csv = "1.1.6"
-dec = "0.4.8"
-derivative = "2.2.0"
-encoding = "0.2.0"
-enum-iterator = "1.2.0"
-fallible-iterator = "0.2.0"
-hex = "0.4.3"
-hmac = "0.12.1"
-itertools = "0.10.5"
-once_cell = "1.16.0"
-md-5 = "0.10.5"
-mz-lowertest = { path = "../lowertest" }
-mz-ore = { path = "../ore", features = ["network"] }
-mz-pgrepr = { path = "../pgrepr" }
-mz-repr = { path = "../repr" }
-mz-proto = { path = "../proto" }
-num = "0.4.0"
-num_enum = "0.5.7"
-ordered-float = { version = "3.4.0", features = ["serde"] }
-paste = "1.0.11"
-prost = { version = "0.11.3", features = ["no-recursion-limit"] }
-regex = "1.7.0"
-regex-syntax = "0.6.28"
-serde = { version = "1.0.152", features = ["derive"] }
-serde_json = "1.0.89"
-serde_regex = "1.1.0"
-sha1 = "0.10.5"
-sha2 = "0.10.6"
-tracing = "0.1.37"
-uncased = "0.9.7"
-uuid = {version = "1.2.2", features = ["v5"]}
-proptest = { git = "https://github.com/MaterializeInc/proptest.git", default-features = false, features = ["std"] }
-proptest-derive = { git = "https://github.com/MaterializeInc/proptest.git", features = ["boxed_union"]}
+aho-corasick = { workspace = true }
+anyhow = { workspace = true }
+bytes = { workspace = true }
+bytesize = { workspace = true }
+chrono = { workspace = true, default-features = false, features = ["std"] }
+csv = { workspace = true }
+dec = { workspace = true }
+derivative = { workspace = true }
+encoding = { workspace = true }
+enum-iterator = { workspace = true }
+fallible-iterator = { workspace = true }
+hex = { workspace = true }
+hmac = { workspace = true }
+itertools = { workspace = true }
+once_cell = { workspace = true }
+md-5 = { workspace = true }
+mz-lowertest = { workspace = true }
+mz-ore = { workspace = true, features = ["network"] }
+mz-pgrepr = { workspace = true }
+mz-repr = { workspace = true }
+mz-proto = { workspace = true }
+num = { workspace = true }
+num_enum = { workspace = true }
+ordered-float = { workspace = true, features = ["serde"] }
+paste = { workspace = true }
+prost = { workspace = true, features = ["no-recursion-limit"] }
+regex = { workspace = true }
+regex-syntax = { workspace = true }
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true }
+serde_regex = { workspace = true }
+sha1 = { workspace = true }
+sha2 = { workspace = true }
+tracing = { workspace = true }
+uncased = { workspace = true }
+uuid = { workspace = true, features = ["v5"]}
+proptest = { workspace = true, default-features = false, features = ["std"] }
+proptest-derive = { workspace = true, features = ["boxed_union"]}
```

Comment:
If we can define features in addition to the version number in the workspace `Cargo.toml`, I think it makes sense to also mandate `features = ["boxed_union"]` by default (see #16809).

Reply:
Yep, looks like you can specify features on the workspace dependency that get unioned with each package's features.